tool/gravity/cli/journal.go

```diff
@@ -60,20 +60,20 @@ func exportRuntimeJournal(env *localenv.LocalEnvironment, outputFile string) err
 	if err := m.RoBindMount(logDir, journalDir); err != nil {
 		return trace.Wrap(err)
 	}
-
-	cleanup := func(context.Context) error {
+	defer func() {
```

Comment:
This was another bug I introduced when I changed the semantics of the lib/system/signals#InterruptHandler to only run stoppers on interruption (and not always as before) - so this had to migrate into a `deferred` function as it needs to run always.